test/server/lib/search.test.js

```diff
@@ -127,6 +127,31 @@ describe('server/lib/search', () => {
         expect(results2[0].id).to.eq(accountWithDiacritics.id);
       });
 
+      it('when searching with a country filter give correct results', async () => {
+        const collective1Name = 'JHipster Canada';
+        const collective2Name = 'JHipster Sri Lanka';
+        const collective1 = await fakeCollective({ name: collective1Name, countryISO: 'CA' });
+        const collective2 = await fakeCollective({ name: collective2Name, countryISO: 'LK' });
+        const [results1] = await searchCollectivesInDB('JHipster');
```

Comment:
This one is not searching by country, what is it testing?

Reply:
Ah, I added this to check if the search is working properly even with the countries being present in the collectives. But I think this is rather redundant as we test for the search terms in other tests so I've removed this part and only kept the part where it search by country. 👍🏽